### PR TITLE
Test for CASE with value

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/sql/operator/CaseValueTest.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+        SeriesMethod.insertSeriesCheck(s1);
+
+        Series s2 = new Series(entity(), METRIC_NAME_B);
+        s2.addData(new TextSample("2017-01-01T09:30:00.000Z", "a"));
+        s2.addData(new TextSample("2017-01-02T09:30:00.000Z", "b"));
+        s2.addData(new TextSample("2017-01-03T09:30:00.000Z", "c"));
+        s2.addData(new TextSample("2017-01-04T09:30:00.000Z", "d"));
+        s2.addData(new TextSample("2017-01-05T09:30:00.000Z", "e"));
+        s2.addData(new TextSample("2017-01-06T09:30:00.000Z", "f"));
+        SeriesMethod.insertSeriesCheck(s2);
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/operator/CaseValueTest.java]: I believe that you can do only one call of `SeriesMethod.insertSeriesCheck` method.

File: src/test/java/com/axibase/tsd/api/method/sql/operator/CaseValueTest.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+        assertSqlQueryRows("CASE <value> for strings gives wrong result", expectedRows, sqlQuery);
+    }
+
+    /**
+     * 4021
+     */
+    @Test
+    public void testCaseNoFrom() {
+        String sqlQuery = "SELECT CASE WHEN 0 < 1 THEN 1 ELSE 0 END\n";
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/sql/operator/CaseValueTest.java]: What is the reason of the line break?

Comment (vtols) [inline src/test/java/com/axibase/tsd/api/method/sql/operator/CaseValueTest.java]: Strange, I don't know how it appeared here, it is not intended

**Comment by vtols** [comment]
Not quite sure about package for this test

**Pull request merged**